pages/advanced/code-highlighting.mdx

```diff
@@ -9,18 +9,17 @@ To keep the bundle small, only a [subset of languages](https://github.com/Formid
 are included in the syntax highlighter. If you want to add more languages, you can do the following:
 
 1. Run `yarn add prismjs prism-react-renderer` to add dependencies to your Nextra project.
-2. Add the following code to your `pages/_app.js`:
+2. Add the following code to your `pages/_app.jsx`:
```

Comment:
I guess it's better to suggest `jsx` extension for `_app` and `theme.config` since their content includes `jsx` tags